dask_saturn/core.py

```diff
@@ -121,3 +178,62 @@ def close(self):
             raise ValueError(response.reason)
         for pc in self.periodic_callbacks.values():
             pc.stop()
+
+
+def _options():
+    url = urljoin(BASE_URL, "api/dask_clusters/info")
+    response = requests.get(url, headers=HEADERS)
+    if not response.ok:
+        raise ValueError(response.reason)
+    return response.json()["server_options"]
+
+
+def list_sizes() -> List[str]:
+    """Returns a list of the valid size options for worker_size and scheduler size"""
```

Comment:
These are user-facing docstrings, so the should be pep 8

Reply:
Think they should all be following pep 8 now, lmk if there was anything else in particular you wanted to see